statistics/shortest_time_to_get_all_singles.rb

```diff
@@ -28,17 +32,11 @@ def transform(query_results)
       .map do |person_link, results|
         first_competition_date = results[0]["start_date"]
         first_successes = results
-          .select { |result| result["best"] > 0 && Events::OFFICIAL.include?(result["event_id"]) }
```

Comment:
I don't think we need this. By the SQL query above we ensure the person did all official events. If there are more events, let's say magics, then their "first successes" are obviously sooner than skewb first successes.